assets/javascripts/render.js

```diff
@@ -168,7 +168,7 @@ function renderModuleRow(module, snippets) {
       if (title === 'wait_serial') {
         const previewLimit = 120;
         // jshint ignore:start
-        let shortText = textData.replace(/.*# Result:\n*/s, '');
+        let shortText = typeof textData === 'string' ? textData.replace(/.*# Result:\n*/s, '') : '';
```

Comment:
Not sure if this is right, in sese that it might hide the issue. in line https://github.com/os-autoinst/openQA/blob/14d35b3dbecc5ee3a4774b0a786640e4211ef8ef/assets/javascripts/render.js#L139C22-L139C26 there is an assignment. would it be a better idea to check this and log a warning or smth?

Reply:
It is a mitigation not the solution.

Do you suggest something like this:

```suggestion
if (typeof textData !== 'string') {
    console.warn('Unexpected textData value:', textData);
    textData = ''; // Default to an empty string to avoid crashes
}

let shortText = textData.replace(/.*# Result:\n*/s, '');
```

Reply:
This PR is for mitigating the error, so that people are able to see the jobs detail page again. It's AC1. After that we can analyze why that is happening now. That's AC2. It's in the ticket description